frontend/src/components/MeetingRoom/ReportIssue/FeedbackForm/FeedbackForm.tsx

```diff
@@ -60,8 +60,9 @@ const FeedbackForm = ({
   onFileSelect,
 }: FeedbackFormType): ReactElement => {
   const isSmallViewport = useIsSmallViewport();
-  const heightClass = '@apply h-[calc(100dvh_-_240px)]';
-  const widthClass = isSmallViewport ? '@apply w-[calc(100dvw_-_48px)]' : '';
+  // Account for: 64px panel header + 80px toolbar if small viewport or 96px toolbar if normal viewport + (40px submit button + 24px submit button margin)
+  const height = isSmallViewport ? 'calc(100dvh - 208px)' : 'calc(100dvh - 224px)';
+  const width = isSmallViewport ? 'calc(100dvw - 50px)' : '310px';
```

Comment:
60 + 80 = 124
40 + 24 = 64

How do we arrive that the magic literals used here?  Just curious.

Reply:
Great question that others would surely have! I added that into the last commit. Let me know if it's not clear.

Reply:
Naively, seems like we can use variables to calculate these CSS values.  That said, a tech debt item for another day, if we decide to cross that bridge.